packs/chef/actions/apply.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: 'apply'
+runner_type: 'run-remote-script'
+description: 'Performs one-off resource converge on remote hosts.'
+enabled: true
+entry_point: 'chef_apply.py'
+parameters:
+    attributes:
+        description: 'Load attributes from a JSON file or URL'
+        type: 'string'
+    log_level:
+        description: 'Set the log level (debug, info, warn, error, fatal)'
+        type: 'string'
+    why_run:
+        description: "Enable whyrun mode"
+        type: 'boolean'
+    execute:
+        description: "Execute resources supplied in a string"
+        type: string
+    minimal_ohai:
+        description: "Only run the bare minimum ohai plugins chef needs to function"
+        type: boolean
+    recipe_file:
+        description: "Path to the recipe file to be executed (relative to a remote host)"
+        type: string
+    timeout:
```

Comment:
`timeout` is already a remote runner parameter which means you don't (and shouldn't) specify it in the action metadata file unless you want to change the default value and / or make it immutable.

To confirm it - I assume you specified it because you want to increase the default timeout from `60` to `300` seconds?